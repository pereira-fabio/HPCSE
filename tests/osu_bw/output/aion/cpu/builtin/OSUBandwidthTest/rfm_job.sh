#!/bin/bash
#SBATCH --job-name="rfm_OSUBandwidthTest"
#SBATCH --ntasks=2
#SBATCH --ntasks-per-node=2
#SBATCH --cpus-per-task=1
#SBATCH --output=rfm_job.out
#SBATCH --error=rfm_job.err
srun --cpus-per-task=1 /opt/apps/easybuild/systems/aion/rhel810-20250405/2023b/epyc/software/OSU-Micro-Benchmarks/7.5-gompi-2023b/libexec/osu-micro-benchmarks/mpi/pt2pt/osu_bw
