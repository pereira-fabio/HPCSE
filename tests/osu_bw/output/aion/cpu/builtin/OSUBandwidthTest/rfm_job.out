
# OSU MPI Bandwidth Test v7.5
# Datatype: MPI_CHAR.
# Size      Bandwidth (MB/s)
1                      16.56
2                      33.63
4                      66.52
8                     132.95
16                    274.16
32                    544.36
64                   1099.24
128                  1436.11
256                  2875.39
512                  4968.45
1024                 8349.15
2048                13159.69
4096                16512.98
8192                22011.21
16384               12427.10
32768               14025.27
65536               19086.20
131072              21099.54
262144              16838.17
524288              14443.84
1048576             14597.62
2097152             15076.88
4194304             15185.29
